capitalize_decorator.rb

```diff
@@ -1,6 +1,7 @@
 require './correct_nameable'
 class CapitalizeDecorator < CorrectNameable
   def correct_name
-    @person.correct_name[0].upcase + @person.correct_name[1..@person.correct_name.length]
+    name = @person.correct_name.strip
+    name[0].upcase + name[1..name.length]
```

Comment:
- [ ] [OPTIONAL] There is a better way to capitalise the words in ruby and as you'd expect the [method name is capitalize](https://apidock.com/ruby/String/capitalize), example `"hello".capitalize   #=> "Hello"`